src/planwise/client/handlers.cljs

```diff
@@ -76,3 +76,8 @@
  (fn [_ [_ time]]
    (when (= 0 (mod time 1000))
      {:dispatch [:datasets/refresh-datasets time]})))
+
+(rf/reg-event-fx
+ :dialog/delay-dispatch
+ (fn [_ [_ event-vector]]
+   {:dispatch-later [{:ms 150 :dispatch event-vector}]}))
```

Comment:
See my comment in #360. There may be a better workaround for this particular bug. Otherwise, please add a comment here explaining why the delayed dispatch.